qa/tasks/ceph.py

```diff
@@ -80,7 +80,8 @@ def ceph_crash(ctx, config):
     """
 
     # Add logs directory to job's info log file
-    with open(os.path.join(ctx.archive, 'info.yaml'), 'r+') as info_file:
+    if ctx.archive is not None:
+     with open(os.path.join(ctx.archive, 'info.yaml'), 'r+') as info_file:
```

Comment:
could you use 4-space indent here?

Reply:
I didn't because I would have had to rewrap the contained block below and I didn't want to introduce "noise" into the diff.  I can do that for this PR if you prefer.

Reply:
@mdw-at-linuxbox i created #37928 to encapsulate the "noise" in a dedicated commit. probably you could rebase your change on top of it?

Reply:
@mdw-at-linuxbox hi Marcus, i created #38004 as a follow-up. will close it if you want to continue working on this PR.

Reply:
38004 is fine by me.  Sorry for not responding sooner - I ran into some unrelated problems in teuthology which sort of got in the way...